### PR TITLE
Add GitHub actions support

### LINKED ISSUE #11: Add GitHub actions support
- Run unit tests
- Auto deploy to serverless

### PR DESCRIPTION
**Pull request by lululombard**
This closes #11 by adding GitHub actions.
It also adds Sentry support, a `setup.sh` script for systems running bash, and fixes some typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,25 +4,27 @@ Django (Python) website for RedCraft
...
 
+Create your virtual env: `python3 -m venv env`
+And start your venv: `"env/Scripts/activate"`
 Install  requirements: `pip install -r requirements.txt`
-
-Make migration: `python manage.py makmigrations`
+Make migration: `python manage.py makemigrations`
 Migrate DB: `python manage.py migrate`
-
 Compile sass: `python manage.py sass website/static/website/scss/ website/static/website/css/`
```
Comment (Cod3lta) [inline README.md:24]:
```suggestion
Compile sass: `python manage.py sass website/static/website/scss/ website/static/website/css/` --watch
```
For automatic scss file compilation

Comment (lululombard) [reply README.md:24]: Already documented in the section after :)

**Review by Cod3lta — APPROVED** [review]

File: website/views.py [hunk 2]
```diff
@@ -5,21 +5,35 @@
 from django.views import View
 from django.core.exceptions import ObjectDoesNotExist
 from django.urls import reverse
-from django.http import JsonResponse
+from django.http import JsonResponse, HttpResponse
 from user_agents import parse as parse_user_agents
 
+from django.conf import settings
```
Comment (Likyaz) [inline website/views.py:11]: Django import with django import plz

**Review by Likyaz — APPROVED** [review]
For the next time

**Pull request merged**